dnf-plugins/py3query.py

```diff
@@ -254,13 +254,11 @@ def run(self, args):
                 if req in python_versions.keys():
                     deps_of_pkg[req].add(pkg)
 
-        unversioned_wildcards = ['python-*', '[!/]*-python-*', '[!/]*-python']
-        runtime = self.pkg_query.filter(requires__glob=unversioned_wildcards)
-        buildtime = self.src_query.filter(requires__glob=unversioned_wildcards)
         # unversioned_requirers: {srpm_name: set of srpm_names}
         unversioned_requirers = collections.defaultdict(set)
-        for pkg in progressbar(runtime + list(buildtime), 'Getting unversioned requirers'):
-            for require in pkg.requires + pkg.requires_pre:
+        for pkg in progressbar(self.all_unqualified_requires(), 'Getting unversioned requirers'):
+            for require in (pkg.requires + pkg.requires_pre + pkg.recommends +
+                            pkg.suggests + pkg.supplements + pkg.enhances):
```

Comment:
`self.all_unqualified_requires()` takes a lot of time. I have added info messages for querying each type of dependencies, and the log now looks as follows:
```
...
[====================] 3/3 Querying packages with unversioned dependencies: Done!     
[====================] 3906/3906 Processing packages with unversioned dependencies: Done!
...
```